### DISCUSSION TITLE
Missing reminders from a professional shared calendar

### DISCUSSION THREAD

**Discussion by walrus543** [bitfireAT/icsx5#106] [Q&A] [answered]
Hello,
I shared my Outlook pro calendar: `https://outlook.office365.com/owa/calendar/…/reachcalendar.ics`
I added this url on my nextcloud server.
The calendar appears in Davx5 on my Android device. Then I add the link into ICSx5 to enable the synchronisation.

All my pro events have a reminder. None of them is synchronised.
Is it a known problem? Is there something I can do to synchronise the reminders?

Thank you.
Android 11 / Oneplus 3T (ROM based on LineageOS)

**Comment by ArnyminerZ** [comment, marked as answer]
I have tried to create an event from Outlook, and added a reminder for 1 hour before, and this is the ICS output:
```ical
BEGIN:VEVENT
DESCRIPTION:\n
UID:...
SUMMARY:Testing event with reminder
DTSTART;TZID=Romance Standard Time:20230207T120000
DTEND;TZID=Romance Standard Time:20230207T123000
CLASS:PUBLIC
PRIORITY:5
DTSTAMP:20230207T100615Z
TRANSP:OPAQUE
STATUS:CONFIRMED
SEQUENCE:0
LOCATION:
X-MICROSOFT-CDO-APPT-SEQUENCE:0
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MICROSOFT-CDO-ALLDAYEVENT:FALSE
X-MICROSOFT-CDO-IMPORTANCE:1
X-MICROSOFT-CDO-INSTTYPE:0
X-MICROSOFT-DONOTFORWARDMEETING:FALSE
X-MICROSOFT-DISALLOW-COUNTER:FALSE
END:VEVENT
```
There's no reminder, so my first thought is that Outlook does not support reminders in the ICS file format. Without further information, there's nothing we can do, sorry.

**Comment by walrus543** [comment]
@ArnyminerZ actually I also shared the same ics with my ProtonCalendar account and I do have the reminders.
If you need additional information, let me know. 
Actually I think that Proton automatically adds the reminder so it doesn't come from the ics link.

**Reply by ArnyminerZ** [reply]
Can you give some examples of the generated ics? You can include just one event that you know has the reminders. I'm not able to get any reminders from my Office 365 Pro edition. Maybe there's some admin configuration.

**Reply by walrus543** [reply]
````
BEGIN:VEVENT
DESCRIPTION:\n_____________________________________________________________
 ___________________\nRéunion Microsoft Teams\nRejoindre sur votre ordinat
 eur ou application mobile\nCliquez ici pour participer à la réunion<http
 s://teams.microsoft.com/l/meetup-join/19%3ameeting_NzNmYjUwYzYtOGUyNi00N2Q
 0LWFiZj%40thread.v2/0?context=%7b%22Tid%22%3a%22850cdb41
 -b53a-465a-9383-5did%22%3a%223ea896cf-bd48-4685-aef8-c
 a40422%7d>\nOu composer le numéro (audio seulement)\n+33 1 73 00 
 622751330#<tel:+331759\,\,22751330#>   France\, Paris\nID Con
 férence Téléphone: 227#\nRechercher un numéro local<https://dia
 lin.teams.microsoft.com/a2112d18-f902948ddd5?id=22751330> 
 | Réinitialiser le code confidentiel<https://dialin.teams.microsoft.com/u
 sp/pstnconfg>\nClick the help link\, to get help connecting through
  the audio conference.\nPour en savoir plus<https://aka.ms/JoinTeamsMeetin
 g> | Aide<https://meeting> | Options de réunio
 n<https://teams.microsoft.com/meetingOptions/?organizerId=3ea896cf-bd48-46
 85-aef8-ca4041a-9383-5d151e553a4a&threadI
 d=19_meeting_NzNmYjUwYzYtmFhOTA4OGM2@thread.v2&me
 ssageId=0&language=fr-FR>\n_______________________________________________
 _________________________________\n\n
RRULE:FREQ=WEEKLY;UNTIL=2023081000Z;INTERVAL=1;BYDAY=TH;WKST=MO
UID:xxxx
SUMMARY:Stand-up équipe
DTSTART;TZID=Romance Standard Time:20230112T113000
DTEND;TZID=Romance Standard Time:20230112T120000
CLASS:PUBLIC
PRIORITY:5
DTSTAMP:20230212T171701Z
TRANSP:OPAQUE
STATUS:CONFIRMED
SEQUENCE:0
LOCATION:Réunion Microsoft Teams
X-MICROSOFT-CDO-APPT-SEQUENCE:0
X-MICROSOFT-CDO-BUSYSTATUS:BUSY
X-MICROSOFT-CDO-INTENDEDSTATUS:BUSY
X-MICROSOFT-CDO-ALLDAYEVENT:FALSE
X-MICROSOFT-CDO-IMPORTANCE:1
X-MICROSOFT-CDO-INSTTYPE:1
X-MICROSOFT-DONOTFORWARDMEETING:FALSE
X-MICROSOFT-DISALLOW-COUNTER:FALSE
END:VEVENT
````

**Reply by ArnyminerZ** [reply]
Yes, Proton is surely adding the reminder automatically, there are no alerts on this event. You can do the same on ICSx5, just select the option to add a default notification from the subscription settings.

**Reply by walrus543** [reply]
@ArnyminerZ Sorry but I see no option
![screenshot_setup](https://user-images.githubusercontent.com/5938828/218676337-cc49b7bb-3171-4ad8-86db-a39f8f84b834.png)

**Reply by ArnyminerZ** [reply]
Oops, it's in fact an option in the next version 😅. I believe that if you join the beta program, it will be available, @rfc2822 can confirm. Otherwise, please just wait a little bit, the feature is implemented, we are just working on other things before releasing the next version. You can see how it will work [here](https://github.com/bitfireAT/icsx5/issues/1#issuecomment-1335255662)

**Comment by walrus543** [comment]
That's a good news :) If I understood well it will be possible to add a default reminder on all events.
Unfortunately I won't able to use it since I have events and all-day events and I don't want to get a notification 15min before an all-day event (at 11:45pm) :D
